tests/esp32/spi_keep_active.toit

```diff
@@ -0,0 +1,93 @@
+// Copyright (C) 2022 Toitware ApS.
+// Use of this source code is governed by a Zero-Clause BSD license that can
+// be found in the tests/LICENSE file.
+
+/**
+Tests the '--keep_cs_active' flag of the SPI transfer.
+
+Setup:
+Connect pin 18 and 15 with a 330 Ohm resistor. The resistor isn't
+  strictly necessary but can prevent accidental short circuiting.
+
+Make sure that pins 12, 13, and 14 are not connected to anything important. They
+  are used as output.
+*/
+
+import expect show *
+import spi
+import gpio
+import monitor
+
+
+CS ::= 15
+MISO ::= 12
+MOSI ::= 13
+SCK ::= 14
+
+IN_CS /int ::= 18
+
+class DebugChannel:
+  channel_ := monitor.Channel 1
+  receive:
+    val := channel_.receive
+    print "=> $val"
+    return val
+
+  send x:
+    channel_.send x
+
+main:
+  bus := spi.Bus
+      --clock=gpio.Pin SCK
+      --miso=gpio.Pin MISO
+      --mosi=gpio.Pin MOSI
+
+  device := bus.device
+      --cs=gpio.Pin CS
+      --frequency=10_000_000
+
+  in := gpio.Pin IN_CS --input
+
+  2.repeat: | test_iteration/int |
+    print "starting round $test_iteration"
```

Comment:
Maybe drop this - or capitalize for consistency.

Reply:
removed.